### PR TITLE
Nerf L6 firerate

### PR DESCRIPTION
**Pull request by metalgearsloth**
So it actually matches rifles coz its main advantage is supposed to be big clip and caliber.

:cl:
- tweak: Change L6 firerate from 8 to 5 in line with rifles.

### CODE REVIEW

**Comment by GoodWheatley** [comment]
The L6 bundle already costs 40 TC and gives you a gun that's nearly impossible to accurately aim while firing, compared to the C20R which costs 10 less TC and doesn't have an increase in spread during sustained fire. The L6 needs a buff, not a nerf.

**Comment by metalgearsloth** [comment]
If it's so bad why is it picked so much.

**Comment by GoodWheatley** [comment]
> If it's so bad why is it picked so much.

Do you have data to prove this

**Comment by asperger-sind** [comment]
The tradeoff of 40 TC (your entire balance as a nukie or 2 unspent traitor uplinks combined)
makes it balanced to a degree where it is very worth considering not buying it due to the sheer amount of things you're vulnerable to, and other options for the same amount of TC being as good or better, or just plain more fun.

I disagree on the L6 needing a buff as it is quite strong as-is, neither is the aim that bad after gun rework.

> If it's so bad, why is it picked so much.

From my experience it is picked decently well, but not picked that much to base an opinion of how balanced it really is, to be fair it is solely my experience on the matter and could vary for other players.

**Comment by metalgearsloth** [comment]
> > If it's so bad why is it picked so much.
> 
> Do you have data to prove this

forgot to record the several hours of games I watched.

**Comment by deltanedas** [comment]
from my experience, l6 is picked **at least** once per op, sometimes they have multiple people buy em

**Comment by Stealthbomber16** [comment]
usage does not equal viability

l6 gets picked up by beginner ops frequently because it's the easiest thing to spend your money on

**Comment by deltanedas** [comment]
today i was in a admeme 3 man fish ops game
i didnt buy anything until the end, other guys had juggs and l6s
one guy vanished to i dunno where, other stuck with me and had a good trigger finger, only firing at bozos who were dangerous and when i wasnt able to attack
this basically meant infinite ammo and being less of a "spray down med because you fluked" and more of a "i have a ton of ammo hahahaha you cant stop me" thing
then once we had disk, let all the fish out it was great :)

so in short lower firerate = betterer

**Comment by Vaaankas** [comment]
having a fire rate of 5 sucks ass, no

**Comment by AJCM-git** [comment]
we can test this out
